lib/logger.js

```diff
@@ -31,11 +37,13 @@ function Logger(level, filename) {
   const logger = new Winston.Logger({
     level: logLevel,
     levels: javaLogLevels.levels,
+    colors: javaLogLevels.colors,
     transports: [
       new Winston.transports.Console({
         timestamp: true,
         json: Config.get('log:json'),
-        stringify: Config.get('log:json')
+        stringify: Config.get('log:json'),
+        colorize: !Config.get('log.json')
```

Comment:
Good catch. Oddly it still works. Wonder if nconf does both colon and dot notation.